storm-client/src/jvm/org/apache/storm/messaging/netty/Server.java

```diff
@@ -249,6 +250,15 @@ public Object getState() {
             }
         }
         ret.put("enqueued", enqueued);
+        
+        // Report messageSizes metric
+        if (_cb instanceof IMetric) {
+            Object metrics = ((IMetric) _cb).getValueAndReset();
+            if(metrics instanceof Map && !((Map) metrics).isEmpty()) {
```

Comment:
nit: space after the if

Reply:
Also fitting with what we had previously.  It would be nice to allow an empty map, but not do anything on null.  This would match what we do in other places working with IMetrics.